lib/lua/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+project(lua CXX)
```

Comment:
Maybe change the submodule to latest tag https://github.com/lua/lua/tree/v5.4.4 ?